app/javascript/lib/scp-api.js

```diff
@@ -627,7 +691,7 @@ export default async function scpApi(
   }
   if (toJson) {
     const json = await response.json()
-    throw new Error(json.error)
+    throw new Error(json.error || json.errors)
```

Comment:
I've had issues with this in JS tests, so you'll want to be sure to mock this correctly when you get to writing tests for this.